telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/bot/BaseAbilityBot.java

```diff
@@ -106,6 +106,7 @@
     private final String botUsername;
 
     // Ability registry
+    protected final List<AbilityExtension> extensions = new ArrayList<>();
```

Comment:
I find that exposing this (and the pattern of usage) is not optimal. We could keep this a private list and expose a "addExtensions" method that will add the extension.

```java
public addExtension(List extensions) ....`

Reply:
Good point, I also thought about this. Will change it 👍